apple-pie/styles/style.css

```diff
@@ -0,0 +1,76 @@
+* {
+    margin: 0;
+    box-sizing: border-box;
+    text-align: center;
+}
+
+.header {
+    height: 50%;
+    color: white;
+    text-decoration: none;
+    font-family: Arial, Helvetica, sans-serif;
+    background-image: url(../images/apple-pie.jpg);
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
```

Comment:
little note for styling background, for shorter syntax you ca write smth like this:
```
background: white url(../images/apple-pie.jpg) no-repeat center;
```

https://www.w3schools.com/css/css_background_shorthand.asp